fcrepo-http-api/src/main/java/org/fcrepo/http/api/url/HttpApiResources.java

```diff
@@ -123,8 +123,6 @@ private void addNodeStatements(final FedoraResource resource, final UriInfo uriI
 
                 //RDF the serialization
                 final Resource formatRDF = createResource(REPOSITORY_NAMESPACE + key);
-
-                model.add(formatRDF, RDFS_LABEL, key);
                 model.add(format, dcFormat, formatRDF);
```

Comment:
In the absence of a response from @osmandin, I will assume the second triple is not a problem.